UI/GameSettingsScreen.cpp

```diff
@@ -357,7 +366,11 @@ void GameSettingsScreen::CreateViews() {
 
 	CheckBox *deposterize = graphicsSettings->Add(new CheckBox(&g_Config.bTexDeposterize, gr->T("Deposterize")));
 	deposterize->OnClick.Add([=](EventParams &e) {
-		settingInfo_->Show(gr->T("Deposterize Tip", "Fixes small in-texture glitches that may happen when the texture is upscaled"), e.v);
+		if (g_Config.bTexDeposterize == true) {
+			settingInfo_->Show(gr->T("Deposterize Tip", "Fixes small in-texture glitches that may happen when the texture is upscaled"), e.v);
+		} else {
+			settingInfo_->Show(gr->T("Deposterize Tip", "You may seem to have some artifacts in the game if you uncheck that after you have set the Upscale Level"), e.v);
```

Comment:
Decided to remove the second tip, it's just the opposite of the other.

I'm thinking of a better way to display the tips, maybe inline in a smaller font, instead of in a popup.